### PR TITLE
[IMP] l10n_it_edi: Hook to manage partner in invoice import

### PR DESCRIPTION
**Pull request by TonyMasciI**
The need for this change is give the flessibility to add new features. Eg.  same company have multiple plant/office in this case we will have one VAT code and multiple fiscal code.

### CODE REVIEW

**Review by scigghia — APPROVED** [review]

**Review by sergiocorato — APPROVED** [review]

**Review by marcelofrare — APPROVED** [review]

File: addons/l10n_it_edi/models/account_edi_format.py [hunk 1]
```diff
@@ -296,6 +296,32 @@ def _update_invoice_from_binary(self, filename, content, extension, invoice):
                 return self._import_fattura_pa(decoded_content, invoice)
         return super()._update_invoice_from_binary(filename, content, extension, invoice)
 
+    def get_partner_invoice(self, tree, company):
```
Comment (jco-odoo) [inline addons/l10n_it_edi/models/account_edi_format.py:299]:
```suggestion
    def _l10n_it_get_partner_invoice(self, tree, company):
```
We should make the method private and give it a more specific name

Comment (matteoopenf) [reply addons/l10n_it_edi/models/account_edi_format.py]: is it ready now?

**Review by MarcoCalcagni — COMMENTED** [review]
PLS merge

File: addons/l10n_it_edi/models/account_edi_format.py (hunk 1 above)
Comment (SirTakobi) [reply addons/l10n_it_edi/models/account_edi_format.py]: > We should make the method private and give it a more specific name

@jco-odoo this has been done, can you update your review? Thanks!

**Comment by jco-odoo** [comment]
@lordkrandel At first sight, it looks good to me, but could you have a look as well?

**Comment by lordkrandel** [comment]
The refactoring of this function is really needed, so it's a good move in this direction.

Yet, reformatting code is not something we generally do in the same commit, because it makes diffs more complicated (unless we rewrite the logics, so that the comparison is not useful anymore). This time I had to re-join lines just to check that nothing was functionally changed.

Also, the 100 chars limit is not really enforced in the accounting, especially when it makes code less readable.
But I'll let @jco-odoo decide on that.

**Comment by jco-odoo** [comment]
@TonyMasciI Could you just squash the commits?

**Comment by jco-odoo** [comment]
Or maybe I can try taking the PR msg

**Comment by jco-odoo** [comment]
@robodoo r+ rebase-merge

**Comment by jco-odoo** [comment]
@dbkosky fyi

**Pull request closed without merging**